src/shared/dropdown-list-v2/nested-dropdown-list.tsx

```diff
@@ -190,21 +206,21 @@ export const NestedDropdownList = <T,>({
         onRetry?.();
     };
 
-    const handleListItemClick = (index: number) => {
-        setFocusedIndex(index);
-
-        const activeList = searchActive
-            ? filteredListItems
-            : unfilteredListItems;
-        onSelectItem?.(activeList[index]);
+    const handleListItemClick = (itemIndex: number, vIndex: number) => {
+        setVirtuosoIndex(vIndex);
```

Comment:
is it intended for `setFocusedIndex` to be left out here for the click and hover?